ens-names/src/modules/5_ens_jsonl_out.rs

```diff
@@ -0,0 +1,31 @@
+use serde_json::json;
+use substreams::store::{DeltaProto, Deltas};
+
+use crate::pb::ens::v1 as ENS;
+
+#[substreams::handlers::map]
+pub fn ens_jsonl_out(
+    ens_record_deltas: Deltas<DeltaProto<ENS::Domain>>,
+) -> Result<ENS::Lines, substreams::errors::Error> {
+    let mut lines = vec![];
+
+    for delta in ens_record_deltas.deltas {
+        if !delta.key.as_str().ends_with(".eth") {
+            continue;
+        }
+
+        lines = vec![json!({
```

Comment:
@yulesa wanted to get your thoughts on if this is a good output format for a ENS lookup table

Reply:
Ideally we should use a DTO struct here with some derives instead as I think it will lead to cleaner integration for testing and DWH integration (utilizing multiple data output formats without needing to add directly to the code) although all this can come later once a first end-to-end implementation of solid testing and DWH integration have been established.